reactor-client/pom.xml

```diff
@@ -0,0 +1,195 @@
+            <include>com/aerospike/client/listener/*</include>
+            <include>com/aerospike/client/policy/*</include>
+            <include>com/aerospike/client/task/*</include>
+            <include>com/aerospike/client/admin/Privilege.java</include>
+            <include>com/aerospike/client/admin/PrivilegeCode.java</include>
+            <include>com/aerospike/client/admin/Role.java</include>
+            <include>com/aerospike/client/admin/User.java</include>
+            <include>com/aerospike/client/async/AsyncClient.java</include>
+            <include>com/aerospike/client/async/AsyncClientPolicy.java</include>
+            <include>com/aerospike/client/async/EventLoop.java</include>
+            <include>com/aerospike/client/async/EventLoopBase.java</include>
+            <include>com/aerospike/client/async/EventLoops.java</include>
+            <include>com/aerospike/client/async/EventPolicy.java</include>
+            <include>com/aerospike/client/async/IAsyncClient.java</include>
+            <include>com/aerospike/client/async/MaxCommandAction.java</include>
+            <include>com/aerospike/client/async/Monitor.java</include>
+            <include>com/aerospike/client/async/NettyEventLoop.java</include>
+            <include>com/aerospike/client/async/NettyEventLoops.java</include>
+            <include>com/aerospike/client/async/NioEventLoop.java</include>
+            <include>com/aerospike/client/async/NioEventLoops.java</include>
+            <include>com/aerospike/client/async/Throttle.java</include>
+            <include>com/aerospike/client/async/Throttles.java</include>
+            <include>com/aerospike/client/cluster/Node.java</include>
+            <include>com/aerospike/client/cluster/ThreadDaemonFactory.java</include>
+            <include>com/aerospike/client/lua/LuaConfig.java</include>
+            <include>com/aerospike/client/lua/LuaCache.java</include>
+            <include>com/aerospike/client/query/Filter.java</include>
+            <include>com/aerospike/client/query/IndexCollectionType.java</include>
+            <include>com/aerospike/client/query/IndexType.java</include>
+            <include>com/aerospike/client/query/PredExp.java</include>
+            <include>com/aerospike/client/query/RecordSet.java</include>
+            <include>com/aerospike/client/query/ResultSet.java</include>
+            <include>com/aerospike/client/query/Statement.java</include>
+            <include>com/aerospike/client/util/ThreadLocalData.java</include>
+          </sourceFileIncludes>
+        </configuration>
+      </plugin>
+    </plugins>      
+  </build>
+
+  <profiles>
+    <profile>
+      <id>jdk18</id>
+        <activation>
+          <jdk>1.8</jdk>
+        </activation>
+      <build>
+        <plugins>   
+          <plugin> 
+            <artifactId>maven-javadoc-plugin</artifactId>
+            <configuration>
+              <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
maven-javadoc-plugin may be updated to latest version.
it supports: <doclint>none</doclint>